rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/MavenExecutor.java

```diff
@@ -131,8 +131,10 @@ private String getClassPath() {
 	public static void main(String[] args) {
 		if (args.length > 2) {
 			logger.info("Running maven build with settings='{}'", args[2]);
-			String[] split = args[2].split("=");
-			System.setProperty(split[0], split[1]);
+			if (args[2].contains("=")) {
```

Comment:
If this is conditional we should put the log statement from above inside the if-clause.